js/gateio.js

```diff
@@ -2620,7 +2620,8 @@ module.exports = class gateio extends Exchange {
             throw new ArgumentsRequired (this.id + ' fetchOrder() requires a symbol argument');
         }
         await this.loadMarkets ();
-        const isStop = this.safeValue (params, 'isStop', false);
+        const stop = this.safeValue2 (params, 'is_stop_order', 'stop', false);
```

Comment:
Aren't all our parameters camelCase? Like `stopPrice` and `premiumIndex`?

Reply:
In this case `is_stop_order` is the exchange-specific parameter, and `stop` is the unified parameter.